docs/get-started/getting-started-workload-qualification.md

```diff
@@ -0,0 +1,135 @@
+---
+layout: page
+title: Workload Qualification
+nav_order: 7
+parent: Getting-Started
+---
+# Getting Started on Spark workload qualification
+
+The RAPIDS Accelerator for Apache Spark does not support all the features which are supported in Apache Spark.
+If you plan to convert existing Spark workload from CPU to GPU, it is highly recommended that you do gap analysis 
+to figure out if there are any unsupported features such as functions, expressions, data types, data formats.
+After that we will know which workload is more suitable to GPU, which is called "workload qualification" here.
+
+If there are certain operators which can not run on GPU due to the current limitations, they will fallback to CPU mode.
+As a result, it may incur some performance overhead because of host memory <=> GPU memory transfer. 
```

Comment:
Probably should mention that it is not critical to ensure all operations are supported by the GPU and significant performance benefits can be gained before all operations are fully supported by the GPU.  It all depends on how critical the portion that is executing on the CPU is to the overall performance of the query.  For example, if most of the query is on the GPU and a hash aggregation towards the end of the query ends up being on the CPU but only a few thousand rows end up going through the CPU processing, it's not going to be super critical to ensure that last aggregation gets on the GPU because it will contribute so little to the overall query performance.  Obviously running the query with the RAIPDS Accelerator is best, but use the relative row counts and/or data sizes moving through various stages of the query to help estimate the relative importance of having an operation be GPU accelerated or not.

Reply:
I added below words from @jlowe 's suggestion:
```
However it is not critical to ensure all operations are supported by the GPU and significant performance benefits can 
be gained before all operations are fully supported by the GPU. It all depends on how critical the portion that is 
executing on the CPU is to the overall performance of the query.
```
Is that enough?

Reply:
Adding that text should be fine to cover it.